src/elements/content-preview/stories/ContentPreview.stories.js

```diff
@@ -1,54 +1,91 @@
 // @flow
 import * as React from 'react';
+import { text } from '@storybook/addon-knobs';
 import { IntlProvider } from 'react-intl';
 import ContentPreview from '../ContentPreview';
 import notes from './ContentPreview.notes.md';
 
-export const Preview = () => (
-    <IntlProvider locale="en">
-        <ContentPreview features={global.FEATURES} fileId={global.FILE_ID} hasHeader token={global.TOKEN} />
-    </IntlProvider>
-);
+export const Preview = () => {
+    const fileId = text('File ID', global.FILE_ID);
+    const token = text('Access Token', global.TOKEN);
+
+    return (
+        <IntlProvider locale="en">
+            <ContentPreview
+                features={global.FEATURES}
+                fileId={fileId}
+                hasHeader
+                key={`${fileId}-${token}`}
+                token={token}
+            />
+        </IntlProvider>
+    );
+};
 
 export const PreviewWithAnnotations = () => {
+    const fileId = text('File ID', global.FILE_ID);
```

Comment:
Can these be declared once at the top of the module?

Reply:
No, they're supposed to be declared in the story.